### PR TITLE
feat: resolve #473 with aria-controls support for accordion

### LINKED ISSUE #473: [Accordion] TODOs
- add support for the `aria-controls` attr
- add unit tests for the accordion component

### PR DESCRIPTION
**Pull request by jh3y**
#### Description
- Adds support for `aria-controls` by passing an `id` prop to `Panel` #473

#### Scope

- [ ] Patch: Bug Fix
- [x] Minor: New Feature
- [ ] Major: Breaking Change

### CODE REVIEW

File: src/accordion/panel.js [hunk 1]
```diff
@@ -61,7 +61,14 @@ class Panel extends React.Component<PanelPropsT> {
   }
 
   render() {
-    const {expanded, disabled, overrides = {}, children, title} = this.props;
+    const {
+      expanded,
+      id,
```
Comment (gergelyke) [inline src/accordion/panel.js]: should we call this `aria-controls`, if used for that? I believe we follow a practice like that in other components too

Comment (jh3y) [reply src/accordion/panel.js]: Updated 👍

File: src/accordion/__tests__/panel.test.js [hunk 2]
```diff
@@ -37,6 +37,27 @@ describe('Panel', () => {
     );
   });
 
+  test('aria-controls populated when ID provided', () => {
```
Comment (gergelyke) [inline src/accordion/__tests__/panel.test.js]: can you update this too with the aria prop?

File: src/accordion/panel.js [hunk 3]
```diff
@@ -116,7 +125,11 @@ class Panel extends React.Component<PanelPropsT> {
                 overrides={toggleIconOverrides}
               />
             </Header>
-            <Content {...sharedProps} {...contentProps}>
+            <Content
+              {...sharedProps}
+              {...contentProps}
+              {...(ariaControls ? {id: ariaControls} : {})}
```
Comment (gergelyke) [inline src/accordion/panel.js]: do we have to provide the id here, or `aria-controls`?

**Pull request merged**